### PR TITLE
Add support for Statement models and endpoints

### LINKED ISSUE #99: Add support for statements

### PR DESCRIPTION
**Pull request by skovmand**
Fixes #99 

- Adds support for the `Statement` endpoints
- Adds `Statement` and `FiscalYear` models

### CODE REVIEW

**Comment by skovmand** [comment]
@lasseebert Will you review this with me?

File: lib/wise_homex.ex [hunk 1]
```diff
@@ -618,6 +618,17 @@ defmodule WiseHomex do
...
+  # Statement
+
+  @doc """
+  Get a Statement by id
+  """
+  def get_statement(config, id, query \\ %{}), do: api_client().get_statement(config, id, query)
+  def get_statements(config, query \\ %{}), do: api_client().get_statements(config, query)
+  def create_statement(config, attrs, rels), do: api_client().create_statement(config, attrs, rels)
+  def update_statement(config, id, attrs), do: api_client().update_statement(config, id, attrs)
+  def delete_statement(config, id), do: api_client().delete_statement(config, id)
```
Comment (lasseebert) [inline lib/wise_homex.ex]: Perhaps add `@doc`s for all five functions for consistency?

File: lib/wise_homex/models/property.ex [hunk 2]
```diff
@@ -7,6 +7,7 @@ defmodule WiseHomex.Property do
     belongs_to :admin, WiseHomex.Account
     belongs_to :external_info, WiseHomex.ExternalInfo
     has_many :addresses, WiseHomex.Address
+    has_many :statements, WiseHomex.Statement
```
Comment (lasseebert) [inline lib/wise_homex/models/property.ex]: Just a note: A property also has many fiscal_years. We don't need to include that here, though.

Comment (skovmand) [reply lib/wise_homex/models/property.ex]: Let's do that when we get to fiscal years

File: lib/wise_homex.ex (hunk 1 above)
Comment (skovmand) [reply lib/wise_homex.ex]: Added in https://github.com/wise-home/wise_homex/pull/100/commits/d12fda05fd06346a8417498ccc45bf5337ea4211

**Pull request merged**